dts/bindings/adc/adc-controller.yaml

```diff
@@ -50,6 +50,7 @@ child-binding:
           - ADC_GAIN_2_5: x 2/5
           - ADC_GAIN_1_2: x 1/2
           - ADC_GAIN_2_3: x 2/3
+          - ADC_GAIN_2_3: x 3/4
```

Comment:
```suggestion
          - ADC_GAIN_3_4: x 3/4
```

Reply:
fixed this line.

Reply:
Please squash the two commits into one.

Reply:
And correct the commit title so that it starts with "drivers: adc: ".